### ISSUE TITLE
onSuccess / onFailure callbacks?

### ISSUE THREAD

**Issue by giorgio79** [padolsey/findAndReplaceDOMText#46]
Hello,

I would like to execute javascript after the element is highlighted, and it looks like js that comes right after findAndReplaceDOMText does not yet find the added element. Any tips? I am trying to add a qtip on the highlighted element.

**Comment by giorgio79** [comment]
PS looks like it is going to be sg else, as the element exists when I console log it :P

**Comment by giorgio79** [comment]
Hmmm, occasionally the replacement is too slow, and when I try to show a qtip2 on the replaced and wrapped element it is sometime missing...Maybe an onSuccess event would be great after all. :)

**Comment by jkupczak** [comment]
I'd love to be able to get a count of how many replacements happened once my script is done.

**Comment by padolsey** [comment]
@giorgio79 `findAndReplaceDOMText` is completely synchronous, so there's no need for a callback. (i.e. it will always have finished running by the time the next line of JS runs). I think you might be experiencing another issue. (Please post it up to JSFiddle if you'd like help)

@jkupczak Sounds like a reasonable feature. For now, FWIW, you can discern the number of matches via the reverts array, e.g.

```js
var f = findAndReplaceDOMText(...);
console.log(f.reverts.length); // number of reverts will always === number of matches
```
